DQMServices/Components/scripts/dqmiodatasetharvest.py

```diff
@@ -208,13 +151,27 @@ def harvestfile(fname):
         # inclusive range -- for 0 entries, row is left out
         firstidx, lastidx = idxtree.FirstIndex, idxtree.LastIndex
         metree = getattr(f, treenames[metype])
+        metree.GetEntry(0)
         metree.SetBranchStatus("*",0)
         metree.SetBranchStatus("FullName",1)
 
         for x in range(firstidx, lastidx+1):
             metree.GetEntry(x)
             mename = str(metree.FullName)
-            if mename in interesting_mes:
+
+            if mename.find("AlCaReco") != -1: 
+              continue
+
+            if mename.find("Isolated") != -1:
+              continue
+            
+            if mename.find("HLT") != -1:
+              continue
+            
+            if not ((mename.find("SiStrip") >= 0) or (mename.find("OfflinePV") >= 0) or (mename.find("PixelPhase1") >= 0) or (mename.find("Tracking") >= 0 )):    
+              continue
+
+            if check_interesting(mename):
```

Comment:
Do you have any special reasons to apply a veto on "AlCaReco", "Isolated", "HLT" and requiring at least one match among "SiStrip", "OfflinePV", "PixelPhase1", "Tracking"? I expected you specify them directly with "interesting_mes"

Reply:
well, no, I couldn't do it in interesting_mes once added the fnmatch. The point is that AlCaReco, IsolatedBunches and HLT workspaces all contains a SiStrip folder, which is a copy with different selections of the original SIStrip folder in the GUI. Not adding those lines I would collect also the histograms with the same name in the other folders. This could be removed for the general case, I left these on purpose to remind people, there can be cases such this one.